app/services/designation-service.js

```diff
@@ -171,6 +176,108 @@ export default ApiBaseService.extend(NavigationAwareness, {
     return this.PUT(url, params);
   },
 
+  async resolveOrder(order) {
+    if (order) {
+      return order;
+    }
+    return this.userPickOrder({
+      state: ACTIVE_ORDER_STATES.join(",")
+    });
+  },
```

Comment:
```suggestion
  async resolveOrder(order) {
    return order || this.userPickOrder({
      state: ACTIVE_ORDER_STATES.join(",")
    });
  },
```

unimportant, just an alternative